hudi-common/src/main/java/org/apache/hudi/common/config/DFSPropertiesConfiguration.java

```diff
@@ -117,7 +137,50 @@ public void addProperties(BufferedReader reader) throws IOException {
     }
   }
 
+  public static TypedProperties getGlobalConfig() {
```

Comment:
```suggestion
  public static TypedProperties getGlobalProps() {
```
minor: "config" a bit vague. "Props" strongly hints it's a Properties